src/Momento.Sdk.Incubating/Momento.Sdk.Incubating.csproj

```diff
@@ -34,10 +34,9 @@
 		<RepositoryUrl>https://github.com/momentohq/client-sdk-dotnet-incubating</RepositoryUrl>
 	</PropertyGroup>
 	<ItemGroup>
-		<PackageReference Include="Momento.Sdk" Version="0.38.1" />
-		<PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1">
-			<IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
-			<PrivateAssets>all</PrivateAssets>
-		</PackageReference>
+    <PackageReference Include="Momento.Sdk" Version="0.38.2" />
+	</ItemGroup>
+	<ItemGroup>
+	  <ProjectReference Include="..\..\..\client-sdk-dotnet\src\Momento.Sdk\Momento.Sdk.csproj" />
 	</ItemGroup>
```

Comment:
Looks like this got clobbered. We want to keep the SourceLink-GitHub dependency, bump the Momento-Sdk, and not include the temporary reference.